code/game/objects/structures/lattice.dm

```diff
@@ -34,6 +34,13 @@
 	. = ..()
 	. += deconstruction_hints(user)
 
+/obj/structure/lattice/Destroy(force) // so items on the lattice fall when the lattice is destroyed
```

Comment:
Move this to atom_break() or on_deconstruct(). Destroy() must not have any side effects like leaving debris behind